### DISCUSSION TITLE
如何关闭vueCesium的控制台打印信息

### DISCUSSION THREAD

**Discussion by 1557812300** [zouyaoji/vue-cesium#466] [Q&A]
请问下如何关闭vueCesium的控制台打印信息，比如github的地址和点赞信息，如何注释掉，不在控制台里面打印呢，每次都显示有不能关闭，哎呀。

**Comment by zouyaoji** [comment]
你好，开发环境会输出哈，没给去掉参数。但是生产环境下是不会的。

https://github.com/zouyaoji/vue-cesium/blob/dev/packages/vue-cesium/make-installer.ts#L39
